www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+   </td>
+   <td><strong><em>7</em></strong>
+   </td>
+   <td><strong><em>51</em></strong>
+   </td>
+   <td><strong><em>36</em></strong>
+   </td>
+   <td><strong><em>88</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Punjab</em>
+   </td>
+   <td><strong><em>327</em></strong>
+   </td>
+   <td><strong><em>2,021</em></strong>
+   </td>
+   <td><strong><em>11,234</em></strong>
+   </td>
+   <td><strong><em>13,922</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Rajasthan</em>
+   </td>
+   <td><strong><em>451</em></strong>
+   </td>
+   <td><strong><em>6,152</em></strong>
+   </td>
+   <td><strong><em>18,288</em></strong>
+   </td>
+   <td><strong><em>48,302</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Sikkim</em>
+   </td>
+   <td><strong><em>9</em></strong>
+   </td>
+   <td><strong><em>9</em></strong>
+   </td>
+   <td><strong><em>328</em></strong>
+   </td>
+   <td><strong><em>352</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Tamil Nadu</em>
+   </td>
+   <td><strong><em>879</em></strong>
+   </td>
+   <td><strong><em>10,859</em></strong>
+   </td>
+   <td><strong><em>17,807</em></strong>
+   </td>
+   <td><strong><em>48,330</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Telangana</em>
+   </td>
+   <td><strong><em>438</em></strong>
+   </td>
+   <td><strong><em>4,417</em></strong>
+   </td>
+   <td><strong><em>11,700</em></strong>
+   </td>
+   <td><strong><em>6,888</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Tripura</em>
+   </td>
+   <td><strong><em>1</em></strong>
+   </td>
+   <td><strong><em>972</em></strong>
+   </td>
+   <td><strong><em>72</em></strong>
+   </td>
+   <td><strong><em>6,102</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Uttar Pradesh</em>
+   </td>
+   <td><strong><em>2,198</em></strong>
+   </td>
+   <td><strong><em>72,163</em></strong>
+   </td>
+   <td><strong><em>140,462</em></strong>
+   </td>
+   <td><strong><em>176,615</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Uttarakhand</em>
+   </td>
+   <td><strong><em>246</em></strong>
+   </td>
+   <td><strong><em>1,679</em></strong>
+   </td>
+   <td><strong><em>10,425</em></strong>
+   </td>
+   <td><strong><em>21,072</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>West Bengal</em>
+   </td>
+   <td><strong><em>372</em></strong>
+   </td>
+   <td><strong><em>10,612</em></strong>
+   </td>
+   <td><strong><em>3,562</em></strong>
+   </td>
+   <td><strong><em>43,286</em></strong>
+   </td>
+  </tr>
+</table>
+
+# Creating Karmashapes
+
+Karmashapes was a journey into mapping India; it took years of patience, hundreds of emails and months to fully develop. But what exactly _is_ Karmashapes and _how_ was it developed?
+
+![India's placetype structure](images/placetypes.png "India's placetype structure")
+_(above) This image shows the placetype structure in India. The placetypes circled in red (village, habitations, and urban) are the placetypes that Karmashapes catalogs as `locality` records._
```

Comment:
Add new sentence: ` Generally, villages are populated places with less than 5,000 people and urban towns have at least 5,000 and often millions of people.`

Reply:
Do we know where we got this image?

Reply:
Probably from: https://lgdirectory.gov.in/

Reply:
Yeah, its that image with some slight tweaks. I'll add the url and new sentence.